keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Tuning the number of domains in PodTopologySpread
+kep-number: 3022
+authors:
+  - "@sanposhiho"
+owning-sig: sig-scheduling
+status: provisional
+creation-date: 2021-10-28
+reviewers:
```

Comment:
You don't need that many reviewers/approvers :)

Also, damemi is not a KEP approver (yet :))

Reply:
you can leave me out of this one.